src/vm/functions/mod.rs

```diff
@@ -57,10 +62,13 @@ define_named_enum!(NativeFunctions {
     Sha512("sha512"),
     Sha512Trunc256("sha512/256"),
     Keccak256("keccak256"),
+    Secp256k1Recover("secp256k1-recover"),
+    Secp256k1Verify("secp256k1-verify"),
     Print("print"),
     ContractCall("contract-call?"),
     AsContract("as-contract"),
     ContractOf("contract-of"),
+    PrincipalOf("principal-of"),
```

Comment:
Related to previous feedbacks, the functions returning Optional / Response should end with a `?` (`principal-of?`, etc)

Reply:
addressed with 3907ed5